### ISSUE TITLE
Assertion `index_.size() != 0' failed err code: -6

### ISSUE THREAD

**Issue by Taojianchang** [ablab/spades#360]
hello,
I used metaspades.py and got an  err code: -6 after K77. Could please help with the issue ?
Here is the spades.log.

Thanks you !  
[spades.log](https://github.com/ablab/spades/files/3579490/spades.log)

**Comment by asl** [comment]
Hello

Thank you for your interest in SPAdes.

The log reads:
```
 0:00:43.090   756M / 17G   INFO    General                 (pair_info_count.cpp       : 213)   4 paired reads (0.00299108% of all) aligned to long edges\
  0:00:43.093   228M / 17G   INFO    General                 (pair_info_count.cpp       : 357)     Insert size = 297.5, deviation = 10.5, left quantile = 287, right quantile = 308, read length = 150\
```

So, it's highly likely that right reads do not correspond to the left ones. I'd suggest you to double check that.

Also, given the number of reads and assembly size, it's unlikely to be a metagenome.

**Comment by Taojianchang** [comment]
Thanks！





在 2019-09-06 19:44:30，"Anton Korobeynikov" <notifications@github.com> 写道：


Closed #360.
